buildSrc/build.gradle

```diff
@@ -9,4 +9,5 @@ dependencies {
     implementation 'net.minecraftforge:srgutils:0.4.+'
     implementation 'commons-io:commons-io:2.8.0'
     implementation 'com.google.code.gson:gson:2.10'
+    implementation 'org.eclipse.jgit:org.eclipse.jgit:6.5.0.202303070854-r'
```

Comment:
JGit 6.x and above requires Java 11+. Since our tooling runs at the minimum of Java 8, we need to downgrade to JGit 5.x. Please add a comment noting this as well.

Reply:
Not correct. Since the modularity plugin is in use, Gradle requires Java 11 anyways.

Reply:
Huh, I didn't know that. I was drawing on my knowledge that ForgeGradle itself has the minimum bound of Java 8. 

Because of this, we can upgrade to JGit version to 6.x then, @Matyrobbrt.